.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,90 @@
+<!-- Type of change
+Please label this PR with one of the following labels, depending on the scope of your change:
+- bug
+- enhancement
+- breaking change
+-->
+
+## What does this PR do?
+
+<!-- Mandatory
+Explain here the changes you made on the PR. Please explain the WHAT: patterns used, algorithms implemented, design architecture, message processing, etc.
+
+Example:
+  Expose 'xpack.monitoring.elasticsearch.proxy' in the docker environment variables and update logstash.yml to surface this config option.
+  
+  This commit exposes the 'xpack.monitoring.elasticsearch.proxy' variable in the docker by adding it in env2yaml.go, which translates from
+  being an environment variable to a proper yaml config.
+  
+  Additionally, this PR exposes this setting for both xpack monitoring & management to the logstash.yml file.
+-->
+
+## Why is it important/What is the impact to the user?
+
```

Comment:
Thanks for adding the "impact to the user" language.  This should help clarify thinking around user benefits and give us a head start on release notes.